### PR TITLE
Create admin guide

### PR DESCRIPTION
**Pull request by abrennan89**
Part of #3525

### CODE REVIEW

File: docs/eventing/getting-started.md [hunk 1]
```diff
@@ -250,22 +253,25 @@ EOF
...
     ```
 
 1. Make a HTTP request to the broker. To show the various types of events you can send, you will make three requests:
-    - To make the first request, which creates an event that has the `type`
-       `greeting`, run the following in the SSH terminal:
-       ```
+
+    - To make the first request, which creates an event that has the `type` `greeting`, run the following in the SSH terminal:
+
+       ```shell
```
Comment (matzew) [inline docs/eventing/getting-started.md:274]: that's nice to see the curl example in the getting started

File: docs/eventing/getting-started.md [hunk 2]
```diff
@@ -62,7 +61,7 @@ demonstrate how you can configure your event producers to target a specific cons
 1. To deploy the `hello-display` consumer to your cluster, run the following
    command:
 
-     ```
+     ```yaml
      kubectl -n event-example apply -f - << EOF
      apiVersion: apps/v1
      kind: Deployment
```
Comment (matzew) [inline docs/eventing/getting-started.md:67]: is there a reason we use the plain k8s APIs, rather than, say `ksvc`s?

Comment (abrennan89) [reply docs/eventing/getting-started.md:67]: I think the idea is so that you don't need Serving to use the Eventing examples

File: docs/eventing/getting-started.md (hunk 1 above)
Comment (abrennan89) [reply docs/eventing/getting-started.md:274]: I just cleaned things up, so credit goes to whoever wrote it initially 😄

File: docs/admin/metrics/eventing-metrics-api.md [hunk 3]
```diff
@@ -1,19 +1,16 @@
...
-weight: 99
+title: "Knative Eventing metrics API"
+weight: 10
 type: "docs"
+aliases:
+  - /docs/eventing/metrics
 ---
 
-<br>
+Administrators can monitor the Knative Eventing control plane by looking at the metrics exposed by each Knative Eventing component.
```
Comment (vaikas) [inline docs/admin/metrics/eventing-metrics-api.md:9]: I think it would be more data plane?

File: docs/admin/metrics/serving-metrics-api.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
+---
+title: "Knative Serving metrics API"
+weight: 10
+type: "docs"
+aliases:
+  - /docs/serving/metrics/#admin
+---
+
+Administrators can monitor the Knative Serving control plane by looking at the metrics exposed by each Knative Serving component.
```
Comment (vaikas) [inline docs/admin/metrics/serving-metrics-api.md:9]: ditto. data plane.

Comment (vaikas) [reply docs/admin/metrics/serving-metrics-api.md:9]: well :) reading through these, maybe both control and dataplane? Which makes me think if they should be grouped then differently? Thinks like Controller / Webhook vs. Activator. I think some of these are more applicable for the Application developer vs. cluster admin, but just jotting it down and get somebody from serving to help :)

File: docs/admin/metrics/eventing-metrics-api.md [hunk 5]
```diff
@@ -1,19 +1,16 @@
...
 
-## Admin
-
-Administrators can monitor Eventing based on the metrics exposed by each Eventing component.
-Metrics are listed next.
-
-### Broker - Ingress
+### Ingress metrics
 
 Use the following metrics to debug how broker ingress performs and what events are dispacthed via the ingress component.
```
Comment (vaikas) [inline docs/admin/metrics/eventing-metrics-api.md:15]: Not your doing, but while here, could you fix the typo: dispacthed => dispatched? :)

**Review by vaikas — COMMENTED** [review]
Thanks!! Couple of small things.

File: docs/admin/metrics/serving-metrics-api.md (hunk 4 above)
Comment (abrennan89) [reply docs/admin/metrics/serving-metrics-api.md:9]: Yes, always jot it down, this is all good context 😄

Is there a description of control vs dataplane anywhere that we can reuse, or can we create a blurb about each of those to explain the difference in the metrics topics? @skonto is explaining the difference between these something you could help with?

File: docs/admin/metrics/eventing-metrics-api.md (hunk 5 above)
Comment (abrennan89) [reply docs/admin/metrics/eventing-metrics-api.md:15]: 😂 of course, I didn't even notice it, thank you

File: docs/admin/install/check-install-version.md [hunk 6]
```diff
@@ -3,6 +3,8 @@ title: "Checking the version of your Knative components"
 linkTitle: "Checking your install version"
 weight: 20
 type: "docs"
+aliases:
```
Comment (omerbensaadon) [inline docs/admin/install/check-install-version.md:6]: Quick Q: are going to separate the entire site by Admin + Developer Docs? 

I am supportive of this idea, I just want to make sure that's what we're going for considering the sub-folder this currently sits in.

File: docs/admin/logs/_index.md [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+
+You can apply the configuration by entering the command:
+
+```shell
+kubectl apply --filename https://github.com/knative/docs/raw/main/docs/install/collecting-logs/fluent-bit-collector.yaml
+```
+
+The default configuration will classify logs into:
+
+- Knative apps, or pods with an `app=Knative` label
```
Comment (omerbensaadon) [inline docs/admin/logs/_index.md:51]: Should apps be "services" here?

File: docs/admin/logs/_index.md [hunk 8]
```diff
@@ -0,0 +1,176 @@
...
+  [OUTPUT]
+      Name            forward
+      Host            log-collector.logging
+      Port            24224
+      Require_ack_response  True
+```
+
+## Local collector
+
+**NOTE:** This describes a development environment setup, and is not appropriate
```
Comment (omerbensaadon) [inline docs/admin/logs/_index.md:89]: In mkDocs, this would be a great candidate for: 

```
!!! warning
        This describes a development environment setup, ... 
```
        
It would render like this: 

Screen Shot 2021-05-05 at 10 32 01 AM

File: docs/admin/logs/_index.md [hunk 9]
```diff
@@ -0,0 +1,176 @@
...
+for production.
+
+If you are using a local Kubernetes cluster for development, you can create a `hostPath` PersistentVolume to store the logs on your desktop operating system. This allows you to use your usual desktop tools on the files without needing Kubernetes-specific tools.
+
+The PersistentVolumeClaim will look something like this, but the `hostPath` will
+vary based on your Kubernetes software and host operating system. Some example
+values are documented below.
+
+```yaml
```
Comment (omerbensaadon) [inline docs/admin/logs/_index.md:98]: The options below this would be a great candidate for content tabs in mkDocs as well! 

These are rendered: 

```
=== "Kind"

        Some text

=== "Docker Desktop

        Some Text
```

And would render like this: 
Screen Shot 2021-05-05 at 10 36 45 AM

**Review by omerbensaadon — COMMENTED** [review]
/lgtm 

Peppered in a few references to what things would look like in mkDocs 👍🏼 

This is an awesome start to make your Miro 🍝 vision  into a reality.

File: docs/admin/install/check-install-version.md (hunk 6 above)
Comment (abrennan89) [reply docs/admin/install/check-install-version.md:6]: > Quick Q: are going to separate the entire site by Admin + Developer Docs?

Miro 🍝 explains it better but there would basically be multiple categories and we'll link between docs as required:

- Getting started
- Administrator guide
- Developer guide
- CLI guide
- Knative Serving
- Knative Eventing
- FAQs
- Troubleshooting
- Reference guide

This is in flux cos as I said previously I expect this to be a living document, but this I think is a good place to start. You can see details about what I have for each topic so far in the 🍝

Comment (abrennan89) [reply docs/admin/install/check-install-version.md:6]: We seemed to have unanimous agreement in Slack that install goes into the admin guide so I think this is the right place for it.

File: docs/admin/logs/_index.md (hunk 7 above)
Comment (abrennan89) [reply docs/admin/logs/_index.md:51]: I think it's the same difference but I didn't write this, I'd imagine @skonto did maybe, I just cleaned it up, so let's get his opinion before we change any wording relating to components?

File: docs/admin/logs/_index.md (hunk 8 above)
Comment (abrennan89) [reply docs/admin/logs/_index.md:89]: Can you put all this in a guide please instead of on the PR 😄 I'm not gonna be implementing it here yet

File: docs/admin/logs/_index.md (hunk 9 above)
Comment (abrennan89) [reply docs/admin/logs/_index.md:98]: Good - put it in a guide please 😄

Comment (omerbensaadon) [reply docs/admin/logs/_index.md:98]: yes yes just wanted to get you thinking about it :)

File: docs/admin/_index.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+type: "docs"
+---
+
+There are two core Knative components that can be installed and used together or independently to provide different functions:
+
+* [Knative Serving](../serving/): Easily manage stateless services on Kubernetes by reducing the developer effort required for autoscaling, networking, and rollouts.
+
+* [Knative Eventing](../eventing/): Easily route events between on-cluster and off-cluster components by exposing event routing as configuration rather than embedded in code.
+
+These components are delivered as Kubernetes custom resource definitions (CRDs), which can be configured by a cluster administrator to provide default settings for developer-created applications and event workflow components.
```
Comment (evankanderson) [inline docs/admin/_index.md:13]: I think there's also some common configuration options for administrators around telemetry (`config-logging`, `config-observability`) that should probably move to the admin guide:

https://knative.dev/docs/install/collecting-metrics/
https://knative.dev/docs/install/collecting-logs/

(I'm not quite sure what to do with the "here's a local-logs-collection solution" at the end of the second; probably turn it into something real or go back and delete it.)

File: docs/admin/install/_index.md [hunk 11]
```diff
@@ -18,6 +18,7 @@ aliases:
   - /docs/install/knative-with-minishift
   - /docs/install/knative-with-pks
   - /docs/install/any-kubernetes-cluster
+  - /docs/install/
```
Comment (evankanderson) [inline docs/admin/install/_index.md:21]: I think we want a simple "getting started" install in addition to the admin guide; do you want to wait for that to go in and/or lift one from https://github.com/julz/docs/blob/mkdocs/docs/getting-started/getting-started.md ?

@julz FYI for possible merge issues.

File: docs/admin/install/installing-istio.md [hunk 12]
```diff
@@ -2,6 +2,8 @@
 title: "Installing Istio for Knative"
 weight: 15
 type: "docs"
+aliases:
+  - /docs/install/installing-istio
```
Comment (evankanderson) [inline docs/admin/install/installing-istio.md:6]: Not sure we want to do this now, but it might make sense to put this under `docs/admin/integrations/istio.md`, and (TODO, not in this PR) expand it to include how to use an existing Istio install with Knative.

Comment (evankanderson) [reply docs/admin/install/installing-istio.md:6]: By TODO, I mean put the following text after the YAML front-matter:

```markdown
<!-- TODO: add documentation on how to reuse an existing Istio installation -->
```

Comment (evankanderson) [reply docs/admin/install/installing-istio.md:6]: (See also #3524 )

File: docs/admin/_index.md (hunk 10 above)
Comment (evankanderson) [reply docs/admin/_index.md:13]: Nevermind, it looks like you did this. Do you want to mention "common configuration options" as another section here, or just rely on the left-hand nav?

File: docs/admin/logs/_index.md [hunk 13]
```diff
@@ -0,0 +1,176 @@
+---
+title: "Logging"
+weight: 50
+type: "docs"
+aliases:
+  - /docs/install/collecting-logs
```
Comment (evankanderson) [inline docs/admin/logs/_index.md:6]: Do we want to make "logs" a top-level next to "install", add a "configuration" sub-tree to hold all the configuration, or have a "common" sub-tree to hold configuration that's common to Serving and Eventing (off the top of my head, the high-availability configuration, logging, and monitoring)?

(I'm trying to avoid the "too many items at one level in the LHS navigation" problem we have today.)

File: docs/admin/serving/config-ha.md [hunk 14]
```diff
@@ -2,6 +2,8 @@
 title: "Configuring high-availability components"
 weight: 50
 type: "docs"
+aliases:
+  - /docs/serving/config-ha
```
Comment (evankanderson) [inline docs/admin/serving/config-ha.md:6]: I think this documentation actually applies to _both_ serving and eventing; it was done for serving first, but this is a case where our Conway's Law docs structure actually hurt us.

File: docs/admin/serving/webhook-customizations.md [hunk 15]
```diff
@@ -2,6 +2,8 @@
 title: "Exclude namespaces from the Knative webhook"
 weight: 70
 type: "docs"
+aliases:
+  - /docs/serving/webhook-customizations
```
Comment (evankanderson) [inline docs/admin/serving/webhook-customizations.md:6]: Again, this actually applies to both serving and eventing, and maybe deserves a "common" configuration options section.

File: docs/admin/upgrade/upgrade-installation.md [hunk 16]
```diff
@@ -4,6 +4,7 @@ weight: 21
 type: "docs"
 aliases:
   - /docs/install/upgrade-installation
+  - /docs/upgrade/upgrade-installation
```
Comment (evankanderson) [inline docs/admin/upgrade/upgrade-installation.md:7]: Maybe rename this to `upgrade/yaml.md`, and the other to `upgrade/operator.md`?

Comment (evankanderson) [reply docs/admin/upgrade/upgrade-installation.md:7]: I'm also okay if you don't want to have to make a lot of superfluous internal link updates in this PR.

File: docs/eventing/getting-started.md [hunk 17]
```diff
@@ -14,22 +14,21 @@ transport the events.
 ## Creating a Knative Eventing namespace
 
 Namespaces are used to group together and organize your Knative resources.
```
Comment (evankanderson) [inline docs/eventing/getting-started.md:16]: It looks like this is unrelated cleanup?

If you want to do this file on its own, I could probably LGTM this on a single go-round.

**Review by evankanderson — COMMENTED** [review]
Thanks for doing this!

There are a couple places where our docs weren't in optimal places before (that you couldn't have known about, because the doc didn't say); I've left some suggestions on content reorganization that this PR made clear because of that.

Other than that, this looks great, thank you very much!

File: docs/admin/install/_index.md (hunk 11 above)
Comment (abrennan89) [reply docs/admin/install/_index.md:21]: You'll laugh but I was asking Omer yesterday if it would be helpful for me to just pull all of that content into the old/main version of the website as I'm rewriting/moving things, so I'm sort of unofficially doing that to provide editing and cos it'll probably make cutting over the new site easier.
I literally just started yesterday so it's very WIP, but I'm doing it as part of a PR where I wanna also add a developer guide, cos I think it'll be easier to multitask: https://github.com/knative/docs/pull/3531/files

File: docs/admin/install/installing-istio.md (hunk 12 above)
Comment (abrennan89) [reply docs/admin/install/installing-istio.md:6]: Yep, I still think there's too much stuff lumped into the install guide right now, but I didn't want to try to get too granular yet cos I'm going to iterate with more PRs once this is merged. I don't wanna make a PR impossible for folks to review by moving everything at once plus all the editing etc 😄

File: docs/admin/logs/_index.md (hunk 13 above)
Comment (abrennan89) [reply docs/admin/logs/_index.md:6]: Can you add this as a suggestion to the Miro doc please so I can see better what you're saying / how this fits in?
We can follow it up in another PR if we want to move this

File: docs/admin/serving/config-ha.md (hunk 14 above)
Comment (abrennan89) [reply docs/admin/serving/config-ha.md:6]: I'll move it to the top level; admin guide > HA

File: docs/admin/serving/webhook-customizations.md (hunk 15 above)
Comment (abrennan89) [reply docs/admin/serving/webhook-customizations.md:6]: awesome, thanks for letting me know

File: docs/admin/upgrade/upgrade-installation.md (hunk 16 above)
Comment (abrennan89) [reply docs/admin/upgrade/upgrade-installation.md:7]: Yeah, I'd rather not add too many :) I'll add it as a todo comment maybe?

File: docs/eventing/getting-started.md (hunk 17 above)
Comment (abrennan89) [reply docs/eventing/getting-started.md:16]: Yeah it's unrelated clean up. I just see things sometimes as I'm working on something else and add them in.
I'll maybe try to make smaller clean up PRs instead but can't promise I won't fix some things as I notice them 😂

File: docs/admin/_index.md (hunk 10 above)
Comment (abrennan89) [reply docs/admin/_index.md:13]: I don't want to add a bunch of links to maintain really until things settle down a bit with moving stuff, but for now I've added the contents list back to the page; does that work for you?

File: docs/admin/upgrade/upgrade-installation.md (hunk 16 above)
Comment (abrennan89) [reply docs/admin/upgrade/upgrade-installation.md:7]: It actually looks like it's not linked to from anywhere so I updated them to upgrade-yaml and upgrade-operator just to make it clearer

File: docs/admin/metrics/serving-metrics-api.md (hunk 4 above)
Comment (abrennan89) [reply docs/admin/metrics/serving-metrics-api.md:9]: Updated these to say data plane and control plane, can you open a follow up issue to improve these docs maybe please and loop in SMEs so we're not blocking the PR on this?

File: docs/admin/metrics/eventing-metrics-api.md (hunk 3 above)
Comment (abrennan89) [reply docs/admin/metrics/eventing-metrics-api.md:9]: Mention both in next commit, see below comment re follow up

File: docs/admin/install/_index.md (hunk 11 above)
Comment (abrennan89) [reply docs/admin/install/_index.md:21]: Resolving since this is outside the scope of this PR

File: docs/admin/install/installing-istio.md (hunk 12 above)
Comment (abrennan89) [reply docs/admin/install/installing-istio.md:6]: Added comment, resolving for this PR

**Comment by RichieEscarez** [comment]
FYI: This PR build is failing with:
```
1:45:27 PM: Error: Error building site: "/opt/build/repo/content/en/community/contributing/elections/2021-TOC/candidate-dprotaso.md:2:1": failed to unmarshal YAML: yaml: line 2: mapping values are not allowed in this context
```

File: docs/admin/logs/_index.md (hunk 13 above)
Comment (skonto) [reply docs/admin/logs/_index.md:6]: @evankanderson tracing too?

File: docs/admin/logs/_index.md (hunk 7 above)
Comment (skonto) [reply docs/admin/logs/_index.md:51]: I didnt write this part but I think using `services` is more precise, although this label is used in different places and seems a bit misleading eg. serving controller.

**Comment by abrennan89** [comment]
A lot of this work will be negated by the new docs structure and things like the getting started, so I'm just going to start working on the mkdocs branch instead and close this PR.
